lib/views.js

```diff
@@ -418,4 +418,51 @@ exports.doc_summaries_by_id = {
       });
     }
   }
-};
\ No newline at end of file
+};
+
+exports.doc_by_place = {
+  map: function(doc) {
+
+    var emitPlace = function(place) {
+      if (!place) {
+        emit([ '_unassigned' ]);
+        return;
+      }
+      while (place) {
+        emit([ place._id ]);
+        place = place.parent;
+      }
+      return;
+    };
+
+    if (doc._id === '_design/medic') {
```

Comment:
This if block can be removed right, since ddoc.type won't match anything?  Seems to have no purpose?